libs/vkd3d/command.c

```diff
@@ -1515,6 +1517,23 @@ static HRESULT STDMETHODCALLTYPE d3d12_command_allocator_Reset(ID3D12CommandAllo
         TRACE("Resetting command list %p.\n", list);
     }
 
+    if ((pending = atomic_add_fetch(&allocator->outstanding_submissions_count, 0)) != 0)
```

Comment:
don't we have atomic load defined anywhere?

Reply:
We could reuse atomic_load from spinlock.h I guess

Reply:
We might want to clean that up at some point, this works as well for now.